addon/components/new-bourbon-select-field.js

```diff
@@ -38,6 +38,11 @@ export default Component.extend(ClickHandlerMixin, {
 
   didInsertElement() {
     this._super(...arguments);
+
+    if (this.get('value') !== this.get('selection.value')) {
+      this.set('value', this.get('selection.value'))
```

Comment:
where does `selection` come from?

Reply:
that is the selection from below.

Reply:
ah okay, so this covers the case where the option associated with the `value` is disabled when this component is rendered, and so we need to find the next enabled option, and update `value`?

Reply:
yup.